index.html

```diff
@@ -148,11 +148,11 @@ <h3 class="panel-title">wee jh</h3>
           <div class="panel panel-default">
             <div class="panel-heading">
               <!--  Add your name here -->
-              <h3 class="panel-title">Your Name Here</h3>
+              <h3 class="panel-title">Roy Koh</h3>
             </div>
             <div class="panel-body">
               <!-- Add your image path in the src attribute -->
-              <img src="img/man.png" alt="man" class="profile-photo img-rounded">
+              <img src="img/roykoh.jpg" alt="man" class="profile-photo img-rounded">
             </div>
           </div>
         </div>
```

Comment:
Please copy the original template and make your changes on the copy, so others still have the template as reference. :+1: